estate/__manifest__.py

```diff
@@ -0,0 +1,7 @@
+{
+    'name': 'My first module',
+    'depends': [
+        'base',
+    ],
+    "application": True,
+}
```

Comment:
![image](https://user-images.githubusercontent.com/35231827/236995222-086ff321-0e7f-4b10-aee7-bd3d579dd4e0.png)

Final new line is missing
https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/missing-final-newline.html